### PR TITLE
Components-UI

### PR DESCRIPTION
**Pull request by gabyse1**
## Changes to the project
- Delete all unuseful files.
- Create *calculator components* in *Calculator.js* file.
- Add styles to components in *Calculator.css* file.
- Add *Calculator component* to *index.js file*.
- Add styles for basic *index.html* file.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
 import reportWebVitals from './reportWebVitals';
+import Calculator from './components/Calculator';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```
Comment (andersonlebon) [inline src/index.js]: - [ ] Nice job :clap:  on making the `calculator.js` a class-based component, however before rendering it here in `index.js` I think for better practice it should first pass in the main component(app.js). Kindly add the `app.js` file that should contain the App class-based component.

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @gabyse1 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (gabyse1) [reply src/index.js]: ## Hello @andersonlebon
Thank you for your review.
I am going to work on your feedback.

**Review by Meltrust — APPROVED** [review]
Hi @gabyse1, 

## Great job implementing the last reviewer's requirements.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gabyse1** [comment]
> Hi @gabyse1,
> 
> ## Great job implementing the last reviewer's requirements.
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gabyse1** [comment]
## Hello @Meltrust 
Thank you very much for your review.

**Pull request merged**